src/utils/platform.ts

```diff
@@ -3,7 +3,7 @@ const platform = typeof navigator === 'undefined' ? 'some platform' : navigator.
 
 export const isChromium = userAgent.includes('chrome') || userAgent.includes('chromium');
 export const isThunderbird = userAgent.includes('thunderbird');
-export const isFirefox = userAgent.includes('firefox') || userAgent.includes('librewolf') || isThunderbird;
+export const isFirefox = userAgent.includes('gecko') || isThunderbird;
```

Comment:
I think it's really bad and not correct go target render engines. And that's all because of how messy user-agents have become these days. Because this PR will break functionality chromium, safari, kiwi browser, vivaldi etc.

Because they all include a simple sentence `AppleWebKit/537.36 (KHTML, like Gecko)`, thus this will evaluate to `true` to a lot of browsers, which shouldn't. We want to know which browsers differ in certain implementation, so we can apply workaround. We shouldn't target a render engine(which also just so happens to be on every user-agent on all major browsers).

I cannot accept such patch in this form, and I personally don't like any workarounds either to accomplish the same deal. It's not ideal in the current form, nor in a workaround patched form that targets a render engine.

Reply:
Oh i didn't checked chromium, that's unfortunate. I'll fix this a bit later, thanks for the review !